Specs/Choice/Chinese/BooleanModel.json

```diff
@@ -0,0 +1,285 @@
+  "Results": [
+    {
+      "Text": "没问题",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+        "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "可以",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "可以",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "中",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "中",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "行",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "行",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "好啊",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "好啊",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "好呀",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "好呀",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "好哇",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "好哇",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "好嘞",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "好嘞",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "同意",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "同意",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": true,
+		    "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "不行",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "不行",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": false,
+        "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "不中",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "不中",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": false,
+        "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "不是",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "不是",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": false,
+        "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "不可以",
+  "NotSupportedByDesign": "python,java",
+  "Results": [
+    {
+      "Text": "不可以",
+      "TypeName": "boolean",
+      "Resolution": {
+        "value": false,
+        "score": 1.0
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "拒绝",
```

Comment:
Can you please add some more complex/longer cases? For example, based on some of the longer specs in English.